src/gvars.c

```diff
@@ -1085,6 +1085,21 @@ Obj FuncISB_GVAR (
 #endif
 }
 
+/****************************************************************************
+**
+*F  FuncIS_AUTO_GVAR( <self>, <gvar> )  . . check if a global variable is auto
+*/
+
+Obj FuncIS_AUTO_GVAR (
+    Obj                 self,
+    Obj                 gvar )
+{
+    // check the argument
+    RequireStringRep("IS_AUTO_GVAR", gvar);
+
+    return ExprGVar(GVarName( CONST_CSTR_STRING(gvar) ) ) ? True : False;
```

Comment:
This will misreported TLS vars in HPC-GAP, which are also using `ExprGVar`, putting an `INTOBJ` into it. So more like this:
```C
  Obj expr = ExprGVar(GVarName( CONST_CSTR_STRING(gvar) ) );
  return (expr && !IS_INTOBJ(expr)) ? True : False;
```

Reply:
ooo, good catch. Will fix.